addons/udes_stock/tests/test_stock_quant.py

```diff
@@ -232,6 +234,9 @@ def test_fifo_without_nones(self):
         self.assertEqual(reserved_quant.package_id, self.pack2)
         self.assertEqual(reserved_quant, self.quant2)
 
+    @unittest.skip(
+        "Skp unittest as the _gather does not correctly return the sorted quants based on fifo when package ids have a mix of nulls"
+    )
```

Comment:
ideally i think we should create an issue for that as it could bite us in the ass later, and then ref the issue in this skip string

Reply:
oh wait the issue already exists https://taiga.unipart.digital/project/udes-x/issue/782
can we just ref this one and update the issue with what you found?